docs/v6_upgrade.md

```diff
@@ -107,7 +107,9 @@ Example going to a specific version:
 
 18. Try running `RAILS_ENV=production bin/rails assets:precompile`. If all goes well, don't forget to clean the generated assets with `bin/rails assets:clobber`.
 
-19. Try your app!
+19. Run `yarn add webpack-dev-server @webpack-cli/serve` if those are not already in your dev dependencies.
```

Comment:
Why would we need `@webpack-cli/serve`?

Reply:
![image](https://user-images.githubusercontent.com/1118459/142079181-e731419e-00bf-4329-8f37-35ca175e1750.png)

Something new. I'll remove.